### DISCUSSION TITLE
Need help with application

### DISCUSSION THREAD

**Discussion by esauflores** [community/community#47262] [GitHub Education]
I have problems with the github application, already applied twice but unabled to verify, it says "The image you selected does not appear to contain your school name.", the document is in spanish, but in the last application i added translated notes to every element (where is the school name, date and my name) , included the school name, i'm a highschool student, what can i do

**Comment by caitlon** [comment]
Hi, 
try maybe to take a picture of your isic (if you have one) instead of sending document, isic has the university name on it.

**Reply by esauflores** [reply]
Thanks for replying, but my country, El Salvador, doesn't have those cards

**Reply by jge162** [reply]
Do you have transcripts for your school? They have to be in English to be readable is all. Try with transcripts.

**Reply by esauflores** [reply]
My school doesn't have transcripts in english, my proof is a receipt of the first payment (month) in the school, it appears all the requested data in spanish, i made notes in english for the information, but i will try, thanks

**Reply by jge162** [reply]
Good luck to you. get it professionally translated. Not you doing it.

**Comment by Cfyye** [comment]
How can I get it off my phone

**Comment by samus-aran** [comment]
Hi @esauflores 👋 

Let's see how we can assist with the following tips:

1. Ensure that the school name that you have **selected** matches the school name you have on the evidence. If it does not then re-select the correct school from the verification map.

2. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope these help, let us know how you get on 👍
